### PR TITLE
docs(migration): update info about local windows event source

### PR DESCRIPTION
**Pull request by sumo-drosiek**

### CODE REVIEW

File: docs/migration.md [hunk 1]
```diff
@@ -2473,7 +2474,8 @@ See [this document](comparison.md#host-metrics) to learn more.__
 
 ### Local Windows Event Log Source (LocalWindowsEventLog)
 
-Local Windows Event Log Source is not supported by the OpenTelemetry Collector.
+There is no migration process from Installed Collector to OpenTelemetry Collector.
+In order to use OpenTelemetry Collector, dedicated app needs to be installed.
```
Comment (aboguszewski-sumo) [inline docs/migration.md:2478]: What do you mean here by `dedicated app`?

Comment (sumo-drosiek) [reply docs/migration.md:2478]: I don't think there is any documentation yet I can relate to.

File: docs/migration.md [hunk 2]
```diff
@@ -2306,7 +2306,8 @@ processors:
 
 ### Local Windows Event Log Source
 
-Local Windows Event Log Source is not supported by the OpenTelemetry Collector.
+There is no migration process from Installed Collector to OpenTelemetry Collector.
+In order to use OpenTelemetry Collector, dedicated app needs to be installed.
```
Comment (perk-sumo) [inline docs/migration.md:2310]:
```suggestion
In order to use OpenTelemetry Collector, dedicated Sumo Logic app needs to be
installed.
```

File: docs/migration.md (hunk 1 above)
Comment (perk-sumo) [reply docs/migration.md:2478]:
```suggestion
In order to use OpenTelemetry Collector, dedicated Sumo Logic app needs to be
installed.
```

**Review by perk-sumo — APPROVED** [review]
LGTM pending two comments

File: docs/migration.md (hunk 1 above)
Comment (aboguszewski-sumo) [reply docs/migration.md:2478]: @perk-sumo yes, that's what I meant

**Pull request merged**